Ported/combat-bees/Assets/Scripts/Components/BaseComponents.cs

```diff
@@ -2,5 +2,10 @@
 using Unity.Mathematics;
 struct Velocity : IComponentData
 {
-    public float3 velocity;
+    public float3 Value;
```

Comment:
I guess we go for the Value naming then ^^

Reply:
Yep.